BrainPortal/app/controllers/neurohub_application_controller.rb

```diff
@@ -82,6 +82,12 @@ def check_if_locked
 
   end
 
+  # For the moment on the NeuroHub side, we bypass checking the site-wide
```

Comment:
This change was part of a push to master that I made a few days ago.... it's fine it will merge ok.